presto-main/src/test/java/com/facebook/presto/operator/scalar/queryplan/TestJsonPrestoQueryPlanFunctionUtils.java

```diff
@@ -46,21 +46,36 @@ private TestJsonPrestoQueryPlanFunctionUtils() {}
                     "         \"highValue\" : \"NaN\",\n" +
                     "         \"nullsFraction\" : 1.0,\n" +
                     "         \"averageRowSize\" : 0.0,\n" +
-                    "         \"distinctValuesCount\" : 0.0\n" +
+                    "         \"distinctValuesCount\" : 0.0,\n" +
+                    "         \"histogram\" : {\n" +
+                    "           \"@class\" : \"com.facebook.presto.cost.UniformDistributionHistogram\",\n" +
```

Comment:
If the default is false, why do we see a plan change here?

Reply:
This was changed in the original PR before making the histogram field optional. So, when the histogram _wasn't_ optional, this test failed because it couldn't deserialize the JSON plan into the correct types. However, with the commit making the type optional, this can probably be reverted.

Reply:
I've reverted this in the commit that makes the field optional